pkg/kubelet/dockertools/docker_manager_linux.go

```diff
@@ -40,8 +40,13 @@ func getContainerIP(container *dockertypes.ContainerJSON) string {
 func getNetworkingMode() string { return "" }
 
 // Returns true if the container name matches the infrastructure's container name
-func containerProvidesPodIP(name *KubeletContainerName) bool {
-	return name.ContainerName == PodInfraContainerName
+func containerProvidesPodIP(containerName string) bool {
```

Comment:
@caseydavenport This was for Windows.  If you look at docker_windows.go, it looks like every non-infra container on Windows could receive an IP address.  But since Kubernetes only allows one IP address per pod, its very unclear which container's IP gets used.

GetPodStatus() appears to just use the last container in the list for the pod IP for Windows.  I'll try to ask @alexbrand about this though...